extra-dev/packages/coq-mathcomp-classical/coq-mathcomp-classical.dev/opam

```diff
@@ -23,17 +23,19 @@ depends: [
 ]
 
 tags: [
-  "category:Mathematics/Logic/Classical logic"
-  "keyword:classical logic"
+  "category:Mathematics/Classical Logic"
```

Comment:
`Mathematics/Logic/Classical logic` is the correct classification, no need to change it.

And `keyword:classical logic` is perfectly fine too (already used by 30 packages).

Reply:
Good catch. Looks like your change in #2731 was never backported upstream (which means we get the wrong keywords in all releases since then).
Fixed upstream: https://github.com/math-comp/analysis/pull/1386